android/src/main/kotlin/com/hypertrack/sdk/flutter/HyperTrackPlugin.kt

```diff
@@ -86,9 +86,9 @@ public class HyperTrackPlugin(): FlutterPlugin, MethodCallHandler, StreamHandler
       "isRunning" -> result.success(sdk.isRunning)
       "start" -> start(result, sdk)
       "stop" -> stop(result, sdk)
-      "addGeotag" -> addGeotag(call.arguments()!!, result, sdk)
+      "addGeotag" -> addGeotag(call.arguments(), result, sdk)
```

Comment:
does `addGeotag` take an optional? what happens if result of `call.arguments()` is `null`?